js/bootstrap-datepicker.js

```diff
@@ -1200,6 +1200,8 @@
 					this.fill();
 					if (this.picker.is(':visible')){
 						e.preventDefault();
+						e.stopPropagation && e.stopPropagation();
+						e.cancelBubble = true; // IE6,7,8 support
```

Comment:
Yes please replace the check of `e.stopPropagation` and the `e.cancelBubble = true` with a single  `e.stopPropagation()` call